tests/create_all/main.tf

```diff
@@ -0,0 +1,31 @@
+module "state_bucket" {
+  source = "git::https://github.com/plus3it/terraform-aws-tardigrade-s3-bucket.git?ref=5.0.0"
+
+  bucket        = "test-bucket-for-ackend"
+  force_destroy = true
+  versioning    = "Enabled"
+
+  public_access_block = {
+    block_public_acls       = true
+    block_public_policy     = true
+    ignore_public_acls      = true
+    restrict_public_buckets = true
+  }
+
+  server_side_encryption_configuration = {
+    bucket_key_enabled = true
+    sse_algorithm      = "aws:kms"
+    kms_master_key_id  = null
+  }
+
+}
+
+resource "aws_dynamodb_table" "this" {
```

Comment:
This shouldn't be here. Instead, use the input object `var.backend_config.dynamodb_table` to have the module create the table.